packages/core/upload/admin/src/components/EditAssetDialog/tests/RemoveAssetDialog.test.tsx

```diff
@@ -105,8 +105,7 @@ const renderCompo = (handleCloseSpy = jest.fn()) => {
           </NotificationsProvider>
         </IntlProvider>
       </DesignSystemProvider>
-    </QueryClientProvider>,
-    { container: document.getElementById('app') as HTMLElement }
+    </QueryClientProvider>
```

Comment:
tried to check the testing library here to understand what was the "container" prop doing before, would you mind sharing me 2 cents about what it is? 🙏

Reply:
basically it was an old way to inject the all the content in the QueryClientProvider in the div wrapper with app id in the body of the html